### PR TITLE
feat(pingcap/tidb-tools): add prow style job with jenkins runner

### PR DESCRIPTION
**Pull request by wuhuizuo**
Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: jobs/pingcap/tidb-tools/latest/pull_verify.groovy [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        daysToKeep(30)
+    }
+    parameters {
+        // Ref: https://docs.prow.k8s.io/docs/jobs/#job-environment-variables
+        stringParam("BUILD_ID")
+        stringParam("PROW_JOB_ID")
+        stringParam("JOB_SPEC")
+    }
+    properties {
+        githubProjectUrl("https://github.com/pingcap/tidb-binlog")
```
Comment (wuhuizuo) [inline jobs/pingcap/tidb-tools/latest/pull_verify.groovy:14]:
```suggestion
        githubProjectUrl("https://github.com/pingcap/tidb-tools")
```

File: jobs/pingcap/tidb-tools/latest/pull_verify.groovy [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        stringParam("JOB_SPEC")
+    }
+    properties {
+        githubProjectUrl("https://github.com/pingcap/tidb-tools")
+    }
+
+    definition {
+        cpsScm {
+            lightweight(true)
+            scriptPath("pipelines/pingcap/tidb-binlog/latest/pull_verify.groovy")
```
Comment (wuhuizuo) [inline jobs/pingcap/tidb-tools/latest/pull_verify.groovy:20]:
```suggestion
            scriptPath("pipelines/pingcap/tidb-tools/latest/pull_verify.groovy")
```

File: prow-jobs/pingcap/tidb-tools/presubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# struct ref: https://pkg.go.dev/k8s.io/test-infra/prow/config#Presubmit
+presubmits:
+  pingcap/docs:
+    - name: pingcap/tidb-tools/pull_verify
+      agent: jenkins
+      decorate: false # need add this.
+      always_run: false
+      skip_report: false
+      optional: false
```
Comment (purelind) [inline prow-jobs/pingcap/tidb-tools/presubmits.yaml:9]:
```suggestion
      optional: true # change this after test
```

File: pipelines/pingcap/tidb-tools/latest/pull_verify.groovy [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+                        find /tmp/tidb_tools_test -name "*.log" | xargs -I {} bash -c 'echo "**************************************"; echo "{}"; cat "{}"'
+                        echo "**************************************"
+                        echo "******************sync_diff.log********************"
+                        cat /tmp/tidb_tools_test/sync_diff_inspector/output/sync_diff.log
+                        echo "********************fix.sql********************"
+                        find /tmp/tidb_tools_test/sync_diff_inspector/output/fix-on-tidb -name "*.sql" | xargs -I {} bash -c 'echo "**************************************"; echo "{}"; cat "{}"'
+                        '''
+                }
+            }
+        }
```
Comment (purelind) [inline pipelines/pingcap/tidb-tools/latest/pull_verify.groovy:111]:
```suggestion
```
Integration testing takes a lot of time and requires a separate pipeline to be created.

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**